feat: custom error page (#65)
moxystudio/next-with-moxy: docusaurus/docs/welcome/instructions.md

    Remove the `themeConfig.algolia` key from `docusaurus/docusaurus.config.js`. The reason is that the search results indexed by Algolia will become out of sync with the documentation, causing it to possibly return wrong results (404 pages).

2. Prepare the rest of the documentation website to be deliverable to your clients!

## 5. Customize the error page

> The boilerplate supports an unstyled error page that is able to distinguish between `Page Not Found` and `Internal Server Error` (also applies to client side errors). It contains appropriate copy for each case, and a link to `/`.

Reviewer: ```suggestion
The boilerplate supports an unstyled error page that is able to distinguish between `Page Not Found` and `Internal Errors` (both server and client errors). It contains appropriate copy for each case, and a link to `/`.
```

> ### Adapting the `<ErrorPage />` component

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```

> You should update this page to fit in with your project's image. To do so, you will need to update the following files:

Reviewer: ```suggestion
You should adapt the `<ErrorPage />` component to fit your project's image. To do so, you will need to update the following files:
```
